tests/projectq_backend_test.py

```diff
@@ -202,3 +202,13 @@ def test_backend_info() -> None:
     assert backend_info.name == "ProjectQBackend"
     assert backend_info.architecture is None
     assert projectq_backend.characterisation == dict()
+
+
+def test_default_pass_serialization() -> None:
+    projectq_backend = ProjectQBackend()
+
+    for opt_level in range(3):
+        default_pass = projectq_backend.default_compilation_pass(opt_level)
+        reconstructed_pass = BasePass.from_dict(default_pass.to_dict())
```

Comment:
Thanks, could make this slightly more efficient by just constructing `default_pass.to_dict()` once.

Reply:
Thanks [9e10373](https://github.com/CQCL/pytket-projectq/pull/89/commits/9e1037316d048e8111be1af84f002691ae222b82)